react-native-snap-carousel/src/SnapCarousel.bs.js

```diff
@@ -0,0 +1,2 @@
+// Generated by BUCKLESCRIPT VERSION 5.0.6, PLEASE EDIT WITH CARE
```

Comment:
the bs.js files should be ignored no? @fakenickels

Reply:
Yes they should!